wrappers/wine-msvc.sh

```diff
@@ -46,10 +46,11 @@ while [ $# -gt 0 ]; do
 	shift
 done
 
-unixify_path='s/\r// ; s/z:\([\\/]\)/\1/i ; /^Note:/s,\\,/,g'
+WINE_MSVC_STDOUT_SED='s/\r//;'"$WINE_MSVC_STDOUT_SED"
```

Comment:
For the frontend scripts that don't set `WINE_MSVC_STDOUT_SED`, we end up with a trailing `;` here without any extra regex following it; is that ok, or should we only add the `;` if `WINE_MSVC_STDOUT_SED` is nonempty here?

We should probably add similar tests for e.g. ml64 and armasm too. I can look into tests for the arm tools at least.

Reply:
I think trailing `;` is ok, otherwise we had already observed some errors from `rc`.

Reply:
I don't have usecases for other tools, no idea what kind of tests should be added now.